docs/source/etl/merge.rst

```diff
@@ -0,0 +1,92 @@
+.. _merge-datasets:
+
+========================
+Merging Variant Datasets
+========================
+
+.. invisible-code-block:: python
+
+    import glow
+    glow.register(spark)
+    path1 = 'test-data/vcf-merge/HG00096.vcf.bgz'
+    path2 = 'test-data/vcf-merge/HG00097.vcf.bgz'
+
+You can use Glow and Spark to merge genomic variant datasets from non-overlapping sample sets into
+a multi-sample dataset. In these examples, we will read from VCF files, but the same logic works
+on :ref:`variant-data <DataFrames backed by other file formats>`.
```

Comment:
This currently reads as `variant-data` - I think you reversed the hyperlink ordering.